### ISSUE TITLE
Opening RTP stream via SDP file sets IsLivestream to false

### ISSUE THREAD

**Issue by Arestihub** [unosquare/ffmediaelement#422]
Opening RTP stream via SDP file sets IsLivestream to false

## Issue Categories

 - [ ] Bug
 - [ ] Feature Request
 - [ ] Question
 - [c ] Not sure

## Version Information

 - [x ] NuGet Package 4.1.310
 - [ ] Build From *Master* branch, Commit *Enter commit id. Example: 704c482*
 - [ ] Build from *Branch Name*, Commit *Enter commit id. Example: 704c482*

## Steps to Reproduce

1. Open stream from an sdp file (element.Open(new Uri(sdpFileName)) or element.source = new Uri(sdpFileName))
2. Check stream status on initialize or media opening event -> IsLivestream is always false

I am not sure what parts of the process this affects, but it seems like it might be a bug. Is there a way to force IsLivestream to be true?

**Comment by mariodivece** [comment]
IsLiveStream is defined as:
`public bool IsLiveStream => IsNetworkStream && IsStreamSeekable == false;`

IsNetworkStream comes from 
`InputContext->iformat->read_play.Pointer != IntPtr.Zero`
accoring to ffmpeg documentation, this is only non-null when the streams are network based. SDP by definition is a session descriptor, implying a network session (i.e. ffmpeg determines this correctly)

And IsStreamSeekable is set to false when the media duration is indeterminate.

In short, IsLiveStream will be set to true when the input format is a network stream according to ffmpeg, and when ffmpeg can't determine the duration of the media being opened then it obviously isn't seekable.

Since I don't have the SDP file or more details of what you want to do, I can't help you more.
